Sample.java

```diff
@@ -1,5 +1,11 @@
 // Time Complexity :
+//Problem_1 : O(max(nums))
+//Problem_2 : O(m*n)
+
```

Comment:
Please recheck the space for delete and earn here

Reply:
Since we are using auxiliary space(Array) of max(nums) length, would not the space complexity for delete and earn be O(max(nums)) ?

Reply:
My bad I meant for minimum falling path ...since we are mutating same array